scipy/signal/tests/test_filter_design.py

```diff
@@ -1912,7 +1912,7 @@ def test_lowpass_1000dB(self):
         rs = 1000
         N, Wn = ellipord(wp, ws, rp, rs, False)
         sos = ellip(N, rp, rs, Wn, 'lp', False, output='sos')
-        w, h = sosfreqz(sos)
+        w, h = freqz_sos(sos)
```

Comment:
Maybe leave one `sosfreqz` in here? Will get us one more line of coverage but more importantly make sure it's actually in the expected namespace for backward compat.